modules/contact-form/grunion-contact-form.php

```diff
@@ -1838,12 +1838,15 @@ function __construct( $attributes, $content = null ) {
 		self::$current_form = $this;
 
 		$this->defaults = array(
-			'to'                 => $default_to,
-			'subject'            => $default_subject,
-			'show_subject'       => 'no', // only used in back-compat mode
-			'widget'             => 0,    // Not exposed to the user. Works with Grunion_Contact_Form_Plugin::widget_atts()
-			'id'                 => null, // Not exposed to the user. Set above.
-			'submit_button_text' => __( 'Submit', 'jetpack' ),
+			'to'                     => $default_to,
+			'subject'                => $default_subject,
+			'show_subject'           => 'no', // only used in back-compat mode
+			'widget'                 => 0,    // Not exposed to the user. Works with Grunion_Contact_Form_Plugin::widget_atts()
+			'id'                     => null, // Not exposed to the user. Set above.
+			'submit_button_text'     => __( 'Submit', 'jetpack' ),
+			'customThankyou'         => '', // Whether to show a custom thank you response after submitting a form. '' for no, 'message' for a custom message, 'redirect' to redirect to a new URL.
```

Comment:
Should `Y` be capitalized in "You"?

Reply:
Nope, though I do need to fix the comment! "Thankyou" is the noun ("I give a thankyou to everyone who reviewed this PR"), "thank you" is the verb ("I thank you for reviewing this PR"). 🙂

Reply:
Heh, thank you! :-) I'm often lost with this stuff as non-native, sorry for the false heads up!